migrations/20241129140834_first_migration.sql

```diff
@@ -0,0 +1,48 @@
+-- +goose Up
+-- create enum type "order_status"
+CREATE TYPE "order_status" AS ENUM ('NEW', 'PROCESSING', 'INVALID', 'PROCESSED');
+-- create "users" table
+CREATE TABLE "users" (
+  "id" uuid NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
You need to enable the extension.

Reply:
it exists in schema but by default it's already enabled 


CREATE EXTENSION IF NOT EXISTS pgcrypto; in schema file